modules/applications.js

```diff
@@ -0,0 +1,64 @@
+// Form local storage availability checker function
+
+export function isStorageAvailable(type) {
```

Comment:
- [ ] We should use ES6 syntax for our functions. Kindly refactor this to an arrow function.